app/coffeescript/components/ui/markers.coffee

```diff
@@ -74,6 +75,7 @@ define [
         shadow: shadowPin
         title: @markerTitle(datum)
         datumId: datum.id
+        listingType: listingType
```

Comment:
What do you think about passing the entire `datum`? Then it's agnostic to what a listing is.